utxo.go

```diff
@@ -9,3 +14,51 @@ type UTXO struct {
 	LockingScript *bscript.Script
 	Satoshis      uint64
 }
+
+type utxoJSON struct {
+	TxID         string  `json:"txid"`
+	Vout         uint32  `json:"vout"`
+	ScriptPubKey string  `json:"scriptPubKey"`
+	Amount       float64 `json:"amount"`
```

Comment:
In our other outputJSON under output.go we use 'Value' for the bsv value, we should probably have that match here rather than Amount?